src/microbe_stage/MicrobeStage.cs

```diff
@@ -515,6 +523,9 @@ public void MoveToMulticellular()
             throw new Exception("failed to keep the current scene root");
         }
 
+        // TODO: The multicellular stage needs to be able to track statistics and not break organelle unlocks
+        GameWorld.UnlockProgress.UnlockAll = true;
```

Comment:
Is there code directly in multicellular to unlock all? I think at least freebuild mode for multicellular will need that. And directly debugging starting later stages will be much worse if organelles aren't available and can't be unlocked normally.